server/routes/bookVisit/visitBookedController.ts

```diff
@@ -5,8 +5,23 @@ export default class VisitBookedController {
 
   public view(): RequestHandler {
     return async (req, res) => {
+      const { bookingConfirmed } = req.session
+
+      const prisonPhoneNumber = bookingConfirmed.prisonCode === 'FHI' ? '01234567890' : '09876543210'
+      const prisonWebsite =
+        bookingConfirmed.prisonCode === 'FHI'
+          ? 'https://www.gov.uk/guidance/foston-hall-prison'
+          : 'https://www.gov.uk/guidance/drake-hall-prison'
+
+      const supportedPrison = !!(bookingConfirmed.prisonCode === 'DHI' || bookingConfirmed.prisonCode === 'FHI')
       res.render('pages/bookVisit/visitBooked', {
-        bookingConfirmed: req.session.bookingConfirmed,
+        bookingConfirmed,
+        visitReference: bookingConfirmed.visitReference,
+        prisonName: bookingConfirmed.prisonName,
+        hasPhoneNumber: bookingConfirmed.hasPhoneNumber,
```

Comment:
```suggestion
        bookingConfirmed,
```
just reference the values within the `bookingConfirmed` object within the template - no need to pass them all separately.